src/controllers/posts.ts

```diff
@@ -0,0 +1,54 @@
+import { RequestHandler } from 'express';
+import { Post } from "../models/posts";
+// const users = require('../data/users');
+const posts = require('../../data/posts');
+
+const POSTS: Post[] = posts;
+
+export const getPosts: RequestHandler = (req, res, next) => {
+    res.json({ posts: POSTS })
+}
+
+export const getPostById: RequestHandler<{pid: number}> = (req, res, next) => {
```

Comment:
Since this `{pid: number}` type is reused, have you considered exporting it as its own type?

Reply:
I hadn't.  I will look into that.